web/src/app/administration/workbasket/details/information/workbasket-information.component.ts

```diff
@@ -280,16 +280,16 @@ implements OnInit, OnChanges, OnDestroy {
         response => {
           this.requestInProgressService.setRequestInProgress(false);
           this.workbasketService.triggerWorkBasketSaved();
-          if (response) {
+          if (response.status === 202) {
             // new Key ERROR_TYPES.MARK_ERR
             this.generalModalService.triggerMessage(
-              new MessageModal('There was an error marking workbasket for deletion',
-                'It not possible to mark the workbasket for deletion, It has been deleted.')
+              new MessageModal('Workbasket was marked for deletion.',
+              `The Workbasket ${this.workbasket.workbasketId} still contains completed tasks and could not be deleted. Instead is was marked for deletion and will be deleted automatically as soon as the completed tasks are deleted from the database.`)
             );
           } else {
             // new Key ALERT_TYPES.SUCCESS_ALERT_12
```

Comment:
Please mark this key with a `TODO: message changed` so that I'll find it during my work on TSK-931.

Reply:
This is still not done and important. :)